app/views/games/_gametable.html.erb

```diff
@@ -0,0 +1,33 @@
+<table class="table table table-striped table-hover table-sm">
+  <thead class="table-dark">
+    <tr style="text-align: center">
+      <th scope="col">Number</th>
+      <th scope="col">State</th>
+      <th scope="col">Created</th>
+      <th scope="col">Actions</th>
+    </tr>
+  </thead>
+  <tbody style="text-align: center">
+    <% @games.each do |game| %>
+      <tr>
+        <th scope="row"><%= game.id.split("-").first %></th>
+        <td scope="row"><%= game.state.humanize %></td>
+        <td scope="row"><%= time_ago_in_words(game.created_at) %></td>
+        <td scope="row">
+          <div class="btn-group">
+            <%= link_to game_path(game), class:"text-decoration-none" do %>
+              <i class="bi bi-eye-fill" style="font-size: 1.3rem;"></i>&nbsp;
+            <% end %>
+            <%= link_to edit_game_path(game), class:"text-decoration-none" do %>
+              <i class="bi bi-pencil-fill" style="font-size: 1.3rem;"></i>&nbsp;
+            <% end %>
+            <%= button_to game_path(game), method: :delete, class:"text-decoration-none", class:"btn btn-danger btn-sm" do %>
+              <i class="bi bi-trash-fill"></i>
+            <% end %>
```

Comment:
I don't like that this button is red. I think it should be the same color as the other actions in this table cell

Reply:
I added a line in the issue you created about this icon. It is up to you if you want to fix it in this PR or when you get to that issue